### PR TITLE
Feature setup

### PR DESCRIPTION
**Pull request by Thi-Ponce**
### Lesson Objectives:
- Set up a React app.
- Use React documentation.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    "eslint-plugin-import": "^2.25.4",
+    "eslint-plugin-jsx-a11y": "^6.5.1",
+    "eslint-plugin-react": "^7.29.4",
+    "eslint-plugin-react-hooks": "^4.3.0",
+    "stylelint": "^13.13.1",
+    "stylelint-config-standard": "^21.0.0",
+    "stylelint-csstree-validator": "^1.9.0",
+    "stylelint-scss": "^3.21.0"
+  }
+}
```
Comment (DeliceLydia) [inline package.json]: - [x] Your `package.json` does not have scripts to start a server check if you created your react project well with `npx create-react-app` it should look like this
json

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @Thi-Ponce, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there! Highlights for your project:
- No linter errors 
- Followed git workflow
- Good README


## Required Changes ♻️
_Check the comments under the review._



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

File: package.json (hunk 1 above)
Comment (Thi-Ponce) [reply package.json]: Hey there! Thanks for the review, I did follow the steps on the documentation so I'm not sure what could've cause this error. But I'll take a look!

Comment (Thi-Ponce) [reply package.json]: Hello! The package.json file on the my-app folder is exactly like the one you sent me but if I try to just move them it doesn't work. Do you have any feedback on what I could do?

**Review by mmsesay — APPROVED** [review]
Hi @Thi-Ponce, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**